api/database/migrations/20191115010855-add-review-table.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("HackerReviews", {
```

Comment:
lit -- needs a corresponding model definition in `api/database/models` but lgtm after that

Reply:
should be in the next PR, just wanna make sure it all seems good to you

Reply:
lit yeah this lgtm